### PR TITLE
feat: add `auto-merge-exclude` workflow input

### PR DESCRIPTION
**Pull request by Fdawgs**
Not 100% on if an empty string is acceptable as the default for `excludes` for [fastify/github-action-merge-dependabot](https://github.com/fastify/github-action-merge-dependabot), which is what [string inputs fall back to](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs).

Tackles https://github.com/fastify/workflows/pull/11#issuecomment-1046287233

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
I would put fastify there by default, wdyt?

**Comment by Fdawgs** [comment]
> I would put fastify there by default, wdyt?

Good idea, that's the big dog.

File: README.md [hunk 1]
```diff
@@ -59,6 +58,13 @@ jobs:
       lint: true
 ```
 
+## Inputs
+
+| Input Name            | Required | Type    | Default | Description                                                                        |
+| --------------------- | -------- | ------- | ------- | ---------------------------------------------------------------------------------- |
+| `auto-merge-exclude`  | false    | string  | ``      | Provide a comma separated list of packages that you do not want to be auto-merged. |
```
Comment (mcollina) [inline README.md:65]:
```suggestion
| `auto-merge-exclude`  | false    | string  | `fastify`      | Provide a comma separated list of packages that you do not want to be auto-merged. |
```

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Eomm — APPROVED** [review]

**Pull request merged**